TextToSpeech/impl/TTSSpeaker.cpp

```diff
@@ -398,18 +409,53 @@ void TTSSpeaker::createPipeline() {
     g_object_set(G_OBJECT(m_audioSink), "volume", (double) (m_defaultConfig.volume() / MAX_VOLUME), NULL);
 
     // Add elements to pipeline and link
+     if(m_pcmAudioEnabled) {
+        //Add raw audio caps
+       audiocaps = gst_caps_new_simple("audio/x-raw", "format", G_TYPE_STRING, "S16LE", "rate", G_TYPE_INT, 22050,
+                                "channels", G_TYPE_INT, 1, NULL);
+       if(audiocaps == NULL) {
+	   m_pcmAudioEnabled = false;
+           TTSLOG_ERROR("Unable to add audio caps for PCM audio.");
+           return ;
+       }
+      capsfilter = gst_element_factory_make ("capsfilter", NULL);
+      if (capsfilter) {
+         g_object_set (G_OBJECT (capsfilter), "caps", audiocaps, NULL);
+         gst_caps_unref(audiocaps);
+      }
+      else {
+          m_pcmAudioEnabled = false;
+          TTSLOG_ERROR( "Unable to create capsfilter for PCM audio.");
+          return;
+      }
+     }
     bool result = TRUE;
 #if defined(PLATFORM_BROADCOM)
+    if(!m_pcmAudioEnabled){
```

Comment:
despite the m_pcmAudioEnabled flag, the decodebin is still being created for BRCM